src/components/utils/popover.ts

```diff
@@ -381,30 +480,92 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
       }));
     }
 
+    if (item.isActive) {
+      el.classList.add(Popover.CSS.itemActive);
+    }
+
     return el;
   }
 
   /**
    * Item click handler
    *
    * @param itemEl - clicked item
+   * @param event - click event
    */
-  private itemClicked(itemEl: HTMLElement): void {
+  private itemClicked(itemEl: HTMLElement, event: MouseEvent): void {
```

Comment:
> MouseEvent

and what about Enter keydowns?

Reply:
seems like it should be a PointerEvent anyway

![image](https://user-images.githubusercontent.com/31101125/184319684-0b5d3c16-51d6-4afe-a25d-1ad66e3b76b6.png)